site/static/docs/5.2/assets/brand/OBS-logo-formatted.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="305" height="110">
```

Comment:
Do we need `width` and  `height` + `viewbox` on each formatted logo ?

Reply:
Yep otherwise it is not displayed correctly